projects/igniteui-angular/src/lib/tabs/tab-item.component.ts

```diff
@@ -126,36 +126,6 @@ export class IgxTabItemComponent extends IgxTabItemBase implements AfterViewInit
     @HostBinding('attr.tabindex')
     public tabindex = -1;
 
-    /**
-     * @hidden @internal
-     */
-    @HostBinding('attr.id')
-    public id = 'igx-tab-item-' + this.index;
-
-    /**
-     * @hidden @internal
-     */
-    @HostBinding('attr.aria-label')
-    public ariaLabel = this.label;
-
-    /**
-     * @hidden @internal
-     */
-    @HostBinding('attr.aria-disabled')
-    public ariaDisabled = this.disabled;
-
-    /**
-     * @hidden @internal
-     */
-    @HostBinding('attr.aria-selected')
-    public ariaSelected = this.isSelected;
```

Comment:
You can keep those as they were before since they don't depend on the id attribute.